src/xcsync/Scripts.cs

```diff
@@ -129,7 +129,7 @@ public static List<string> GetFiles (IFileSystem fileSystem, string projPath, st
 		   .ToList ();
 	}
 
-	public static void CopyDirectory (IFileSystem fileSystem, string sourceDir, string destinationDir, bool recursive, bool overwrite = false)
+	public static void CopyDirectory (IFileSystem fileSystem, string sourceDir, string destinationDir, bool recursive, bool overwrite = true)
```

Comment:
This changes the behavior of `CopyDirectory` before this argument was added. This may have unintended side-effects. Instead of changing this default, change the call that needs to overwrite.

Reply:
fixed the recursive CopyDirectory call to honor the existing overwrite value